app/src/main/java/com/jetpack/compose/learning/maps/place/component/MapsPlacePicker.kt

```diff
@@ -0,0 +1,506 @@
+                verticalAlignment = Alignment.CenterVertically
+            ) {
+                items(
+                    items = data.reviews,
+                    key = {
+                        it.time.toString() + it.author_name
+                    },
+                ) {
+                    PlaceReviewItem(it)
+                }
+            }
+        }
+        MapVerticalSpace()
+    }
+}
+
+/**
+ * Single review item. It display user profile photo, name and actual review.
+ */
+@OptIn(ExperimentalCoilApi::class)
+@Composable
+private fun PlaceReviewItem(it: Reviews) {
+    val image = rememberImagePainter(
+        data = it.profile_photo_url,
+        builder = {
+            crossfade(true)
+        }
+    )
+    Card(
+        modifier = Modifier.padding(end = 16.dp),
+        shape = RoundedCornerShape(8.dp),
+        elevation = 3.dp
+    ) {
+        Row(
+            verticalAlignment = Alignment.CenterVertically,
+            modifier = Modifier.padding(16.dp, 8.dp)
+        ) {
+            Image(
+                painter = image,
+                contentDescription = null,
+                modifier = Modifier
+                    .requiredSize(48.dp)
+                    .clip(shape = CircleShape),
+                contentScale = ContentScale.Crop
+            )
+            Spacer(modifier = Modifier.requiredWidth(8.dp))
+            Column {
+                Text(it.author_name, fontWeight = FontWeight.Bold)
+                MapVerticalSpace(5.dp)
+                CompositionLocalProvider(LocalContentAlpha provides ContentAlpha.medium) {
+                    Text(
+                        it.text,
+                        modifier = Modifier.widthIn(
+                            0.dp,
+                            LocalConfiguration.current.screenWidthDp.dp / 1.5f
+                        ),
+                        maxLines = 2,
+                        overflow = TextOverflow.Ellipsis
+                    )
+                }
+            }
+        }
+    }
+}
+
+/**
+ * Single photo item. It display photo of the place.
+ */
+@OptIn(ExperimentalCoilApi::class)
+@Composable
+private fun PlacePhotoItem(it: PhotosItem) {
+    val image = rememberImagePainter(
+        data = it.getPhotoURL(),
+        builder = {
+            crossfade(true)
+        }
+    )
+    Image(
+        painter = image,
+        contentDescription = null,
+        modifier = Modifier
+            .padding(end = 16.dp)
+            .requiredWidth(150.dp)
+            .requiredHeight(100.dp)
+            .clip(shape = RoundedCornerShape(8.dp)),
+        contentScale = ContentScale.Crop
+    )
+}
+
+/**
+ * Utility function to convert search item into annotated item.
+ * This will make searched text bold in result.
+ */
+private fun AutoCompleteItem.annotatedPlaceName(): AnnotatedString {
+    if (matched_substrings.isEmpty()) {
+        return buildAnnotatedString { append(description) }
+    }
+    return buildAnnotatedString {
```

Comment:
Can we use like this?
return buildAnnotatedString {
        append(description)
        if (matched_substrings.isNotEmpty()) {
            matched_substrings.forEach {
                addStyle(
                    style = SpanStyle(fontWeight = FontWeight.Bold, fontSize = 17.sp),
                    start = it.offset,
                    end = it.length + it.offset
                )
            }
        }
    }